cogs/hacking.py

```diff
@@ -0,0 +1,76 @@
+import discord
+from discord.ext import commands
+from utils.geolocation import get_loc_data
+
+
+class Hacking(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+
+    @commands.command()
+    async def iplookup(self, ctx, ipaddr):
+        data = get_loc_data(ipaddr)
+
+        if isinstance(data, dict):
+            try:
+                city = data['city']
+                region = data['region']
+                country = data['country']
+                country_capital = data['country_capital']
+                country_tld = data['country_tld']
+                postal = data['postal']
+                utc_offset = data['utc_offset']
+                currency = data['currency']
+                languages = data['languages'].split(',')
+                asn = data['asn']
+                org = data['org']
```

Comment:
Too many variables here, I'd just use `data['city']` in the f-strings directly, no need to define so many varaibles which are only used once anyway